db/sql/cassandra/README.md

```diff
@@ -7,4 +7,40 @@ The API was tested with Cassandra 3 and supports:
   - net.IPNet can be stored with a MarshalCQL/UnmarshalCQL wrapper go structure
 - dumping all rows except for Table
 - quering by secondary indexes
-- mocking of gocql behavior (using gockle library)in automated unit tests 
+- mocking of gocql behavior (using gockle library)in automated unit tests
+
+# Cassandra Timeouts
+
+The API will allow the client to configure either single node or multi-node cluster.
+Also, the client can configure following timeouts:
+- ConnectTimeout
```

Comment:
what about to change it like this

Connection
QueryTimeout
ReconnectInterval